swift/Podfile.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/swift/Podfile b/swift/Podfile
```

Comment:
can we have two separate config files?

Reply:
that was my first thought, two configs means that we have to update them twice, easily to forget